CMakeLists.txt

```diff
@@ -74,20 +75,6 @@ if(NOT MSVC)
     )
 endif(NOT MSVC)
 
-file(GLOB MARL_TEST_LIST
```

Comment:
Did you mean to remove this? If so, why?

Reply:
It is populated again here: https://github.com/google/marl/blob/ed21a07a1cf612e26f5d5eff9a5440022f1ee377/CMakeLists.txt#L153-L156

Reply:
Well that's a good reason (and is slightly embarrassing). :D

Reply:
Was there not a previous change to remove all of the `/*` source files? i.e would we not prefer the explicit list to the `*_test.cpp`

Reply:
@ben-clayton

Reply:
@AWoloszyn Thanks! That sounds reasonable to me.

Reply:
Yes, d89fe34b68834de7cf9168fd5d1a8a11f14a17cf tried to move away from globs, but the main reason for that change was to avoid using `list(FILTER)` as that not supported by older versions of CMake.
I'd prefer to use an explicit file list over globbing as globs don't automatically detect new files, requiring cmake to be re-run before it is picked up.

That said, I won't block this change for this - its something I can fix up later.